### PR TITLE
[8.0] auth_ldap_populate: backport fix of #1477

### PR DESCRIPTION
**Pull request by damdam-s**
same as #1477

### CODE REVIEW

**Review by sebalix — APPROVED** [review]

**Review by leemannd — APPROVED** [review]
Thanks

File: users_ldap_populate/model/users_ldap.py [hunk 1]
```diff
@@ -100,12 +101,29 @@ def action_populate(self, cr, uid, ids, context=None):
                     conf['ldap_filter'])
             results = self.get_ldap_entry_dicts(conf)
             for result in results:
+                login = result[1][login_attr][0].lower().strip()
```
Comment (tschanzt) [inline users_ldap_populate/model/users_ldap.py:104]: afaik odoo login name is case sensitive wouldn't it be weird to have to log into odoo with lowercase and the rest of the apps with uppercase or am i missing something here? /cc @gurneyalex

**Review by tschanzt — COMMENTED** [review]
looks good. just one question.

File: users_ldap_populate/model/users_ldap.py (hunk 1 above)
Comment (leemannd) [reply users_ldap_populate/model/users_ldap.py:104]: @tschanzt AFAIK ldap is case insensitive, so my guess is that we may receive them in lowercase and want to make sure it stays that way.
EDIT: sidenote, you're totally right on the odoo side

Comment (gurneyalex) [reply users_ldap_populate/model/users_ldap.py:104]: in LDAP, the login field is typically case insensitive, and people are expecting this -> if we need to mimick this in odoo. 

The code here is stolen from auth_ldap, and we need to keep this consistent.

**Review by gurneyalex — APPROVED** [review]

**Review by tschanzt — APPROVED** [review]

File: users_ldap_populate/model/users_ldap.py (hunk 1 above)
Comment (tschanzt) [reply users_ldap_populate/model/users_ldap.py:104]: @leemannd @gurneyalex thanks for the explanation. Makes sense that way.

**Pull request merged**